docs/make.jl

```diff
@@ -1,3 +1,36 @@
+#
+# Usage: from the package root directory, run
+#
+#    julia --proj=docs docs/make.jl
+#
+# To fix doctests instead of merely running them, use
+#
+#   DOCTEST=fix julia --proj=docs docs/make.jl
+#
+# To disable doctests, run
+#
+#   DOCTEST=off julia --proj=docs docs/make.jl
+#
+using Pkg
+Pkg.update()
```

Comment:
This one is debatable I guess, and perhaps indeed clearly bad (as it may e.g. hinder CI). The main reason I put it in is because I feel forgetting to run `update` in your docs environment is a common cause of issues.

Perhaps it should be changed to `instantiate` or entirely dropped. If we drop it, I think we should at least put in language into the comment above that recommends running the `update` manually in that environment on occasion. 

Or, we could keep it, but add a way to control it via a command argument and/or environment var, just like the doctest. (And what the default value would be could be debated).